email.go

```diff
@@ -126,6 +127,12 @@ func NewEmailFromReader(r io.Reader) (*Email, error) {
 			return e, err
 		}
 		switch {
+		case strings.Contains(p.header.Get("Content-Disposition"), "attachment"):
+			attachment, err := parseAttachment(p)
+			if err != nil {
+				return e, err
```

Comment:
shouldn't it be `nil, err`?

Reply:
I agree... but other parts of this function is written as `e, err` by the author... 🤔
I'm just following the convention